api/lib/domain/usecases/accept-organization-invitation.js

```diff
@@ -18,6 +20,14 @@ module.exports = async function acceptOrganizationInvitation({
     throw error;
   }
 
+  if (localeFromCookie) {
+    const user = await userRepository.getById(organizationInvitedUser.userId);
+    user.setLocaleIfNotAlreadySet(localeFromCookie);
+    if (user.mustBePersisted) {
```

Comment:
La propriété `mustBePersisted` a été introduite dans #5914, mais avec du recul elle gagnerait à être renommée de manière plus descriptive qu'impérative en `hasBeenModified` :

```suggestion
    if (user.hasBeenModified) {
```